en/releases/centreon-commercial-extensions.md

```diff
@@ -165,6 +165,21 @@ To use it, it is necessary to install and activate it. Refer to the
 
 ## Centreon MBI
 
+### 20.10.4
+
+`July 15, 2021`
+
+#### Improvements
+
+- Optimized widget requests
+
+#### Bugfixes
+
+- Removed old credentials from the repo
```

Comment:
This is just creds used for tests, it has no impact on the product itself.
Not sure we should put that in the RN.
```suggestion
```